_posts/2019-02-18-bisq-dao-for-bitcoin-maximalists.md

```diff
@@ -56,7 +56,7 @@ However, when there is no entity, as in Bisq's case, things change. New tools ar
 
 Here's a concrete example of how this mechanism allows traders to pay contributors without any central wallets. We use round numbers for simplicity.
 
-Developer does $1000 of work on Bisq software. He requests $1000 of BSQ, and includes 1000 satoshis with his request. DAO stakeholders vote to approve his request, coloring his 1000 satoshis into BSQ worth $1000. Traders buy this BSQ with $1000 of BTC, and then use the BSQ to pay trading fees.
+Developer does $1,000 of work on Bisq software. He requests $1,000 of BSQ and includes 1,000 Satoshis with his request. DAO stakeholders vote to approve his request, coloring his 1,000 Satoshis into BSQ worth $1,000. Traders buy this BSQ with $1,000 of BTC, and then use the BSQ to pay trading fees.
```

Comment:
```suggestion
Developer does $1,000 of work on Bisq software. He requests $1,000 of BSQ and includes 1,000 satoshis with his request. DAO stakeholders vote to approve his request, coloring his 1,000 satoshis into BSQ worth $1,000. Traders buy this BSQ with $1,000 of BTC, and then use the BSQ to pay trading fees.
```

I don't think capitalizing the term "satoshi" makes sense when using it as a unit of account, just like the term "sats" is almost never capitalized since it almost always refers to the unit of account, and how "bitcoin" is used in lowercase when referring to the unit of account (as opposed to "Bitcoin" the protocol).

Reply:
Interesting ... I wasn't sure myself so I just followed what these guys here did:

https://web.archive.org/web/20210909223429/https://en.bitcoinwiki.org/wiki/Satoshi

But when I just checked I also found many examples of the lowercase version, including on the big Wikipedia, so yeah let's just do that instead. It's logical.

https://wikiless.org/wiki/Bitcoin#Units_and_divisibility